src/main/java/br/ufpe/cin/mergers/handlers/singlerenaming/SafeSingleRenamingHandler.java

```diff
@@ -7,20 +7,19 @@
 import br.ufpe.cin.mergers.util.Side;
 import de.ovgu.cide.fstgen.ast.FSTNode;
 import de.ovgu.cide.fstgen.ast.FSTTerminal;
-import org.apache.commons.lang3.tuple.Pair;
 
 import java.util.List;
 
-public class DefaultSingleRenamingHandler implements SingleRenamingHandler {
+public class SafeSingleRenamingHandler implements SingleRenamingHandler {
```

Comment:
tu poderia adicionar um javadoc de classe resumindo em um ou duas linhas o que cada um desses handlers de renaming faz, por exemplo, "This handler only reports a renaming conflict when unstructured merge reports a conflict with the signature of the renamed declaration".

eu vou abrir uma issue pra isso, ai tu manda num outro pull request